uavcan/register/384.Access.1.0.uavcan

```diff
@@ -56,21 +92,24 @@
 #   - default value is contained in the register named "system.parameter=" (optional)
 #
 # The type and dimensionality of the special function registers containing the minimum, maximum, and the default
-# value of a register shall be the same as those of the register.
+# value of a register shall be the same as those of the register they relate to.
 #
 # If a written value exceeds the minimum/maximum specified by the respective special function registers,
 # the server may either adjust the value automatically, or to retain the old value, depending on which behavior
 # suits the objectives of the application better.
 # The values of registers containing non-scalar numerical entities should be compared elementwise.
 #
+#
+#   STANDARD REGISTERS
+#
 # The following table specifies the register name patterns that are reserved by the specification for
 # common functions. These conventions are not mandatory to follow, but implementers are recommended to adhere because
 # they enable enhanced introspection capabilities and simplify device configuration and diagnostics.
 #
 #   REGISTER NAME PATTERN                               TYPE            FLAGS                   RECOMMENDED DEFAULT
 # =====================================================================================================================
 #
-#   uavcan.node.id                                      natural16       mutable, persistent     65535 (unset/PnP)
+#   uavcan.node.id                                      natural16[1]    mutable, persistent     65535 (unset/PnP)
```

Comment:
what's this change about?

Reply:
For clarity. There is no way to represent a standalone scalar using the register interface. We use one-element arrays for that, and this change is intended to make it clearer.